src/book/02-system/02-nusight/01-overview.mdx

```diff
@@ -5,3 +5,101 @@ title: Overview
+
+Each robot displays the number assigned to it by NUsight. The wedge on the representation of the robot indicates the direction of the robot's orientation. The black arcs coming out of each robot indicates the robot's walk path
+
+The orange dot in the playing field indicates where the robot thinks the ball is located. The green arrow coming out of the orange ball indicates how the robot will kick the ball. Usually it should point towards the opponent's goal.
+
+#### Robot Status Display
+
+![Robot Status](./images/nusight-robot-status.png)
+
+This area displays the internal status of a robot, for each robot connected to NUsight. It shows the robot's name, connection status, battery level and the internal state of the robot.
+
+- Mode - Displays the current stage of the game
+- Phase - Displays the current phase of the current game mode
+- Behavior - Displays what the robot is currently doing
+- Penalty - Displays the penalty incurred by the robot, if any
+- Walk Command - Displays the forward/backward velocity, left/right velocity and rotation rate of the robot, in that order.
+- Camera, Ball and Goal Icons
+  - Flash Orange - a new packet is received
+  - Red Outline box - Indicates that a long time has passed since the last packet of data was received.
+  - Greyed Out - Inactive
+
+### Localization
+
+![Localisation](./images/localisation-main.png)
+
+#### Keymapping
+
+| Keyboard | Action                                                         |
+| :------: | -------------------------------------------------------------- |
+|    w     | Move forward                                                   |
+|    s     | Move backward                                                  |
+|    a     | Strafe left                                                    |
+|    d     | Strafe right                                                   |
+|  Space   | Switch between Free Camera, First Person or Thrid Persion view |
+
+|       Mouse       | Action                                          |
+| :---------------: | ----------------------------------------------- |
+| Right mouse click | In first or third person, switch between robots |
+
+### Chart
+
+![Chart](./images/chart-main.png)
+
+Line Chart/2D Scatter - Switch between a line chart or a scatter plot to represent the data on the chart
+
+Right Panel - Select which robot's data to be plotted on the chart
+
+Minimum Value/Maximum Value - set the chart's plotting boundaries
+
+View Seconds - The rate at which nusight reads from the robot's data stream. Lower value means higher frequency of plotting data
+
+### Vision
+
+![Vision](./images/vision-main.png)
+
+This tab can toggle different overlays on the image that is being seen by the robot:
+
+- image
+- compass
+- Horizon
+- Visual Mesh
+- Green Horizon
+- Balls
+- Goals
+
+### Visual Mesh
+
+This section displays the data from the robot's camera and provides a mesh overlay on the image using the Visual Mesh object detection algorithm
```

Comment:
We should get an image for this, maybe @TrentHouliston has one?

Reply:
Can I use the images from, the one with mesh on it https://github.com/Fastcode/VisualMesh/tree/4a64fe3b0b21dacf09d644dee72e9c6d6408a1d3? or should it be the visual mesh inside nusight>

Reply:
That image won't work, what you see in NUsight is a little different. Will try to get an image.